python_ta/transforms/type_inference_visitor.py

```diff
@@ -43,10 +43,13 @@ def set_const_type_constraints(node):
 
 
 def set_tuple_type_constraints(node):
-    # node_types contains types of elements inside tuple.
-    node.type_constraints = TypeInfo(
-        Tuple[tuple(x.type_constraints.type for x in node.elts)]
-    )
+    # If tuple is on RHS, find and set Tuple type
```

Comment:
The "load" context is for more than just the right-hand side of an assignment statement. The key idea here is that the tuple is being evaluated, rather than assigned to.